client/src/components/EventForm.js

```diff
@@ -12,6 +12,7 @@ export default function EventForm({ open }) {
         event_month: '',
         event_day: '',
         event_description: '',
+        eventtags: []
```

Comment:
Maybe you can reconsider the variable names. It is common to us camelCase in javascript
```suggestion
        eventMonth: '',
        eventDay: '',
        eventDescription: '',
        eventTags: []
```